__tests__/AppBaseClientTest.js

```diff
@@ -10,11 +10,14 @@ describe('#AppBase_Client', () => {
       credentials: 'zuGt16TPP:1ede0dc2-e727-476e-bc35-ee2956e750ef',
     });
     expect(client).toEqual({
-      url: 'scalr.api.appbase.io',
-      protocol: 'https',
-      app: 'appbasejs-test-app',
-      credentials: 'zuGt16TPP:1ede0dc2-e727-476e-bc35-ee2956e750ef',
-      headers: {},
+      url: "scalr.api.appbase.io",
+      protocol: "https",
+      app: "appbasejs-test-app",
+      credentials: "zuGt16TPP:1ede0dc2-e727-476e-bc35-ee2956e750ef",
+      headers: {
+        "X-Search-Client": "Appbase JS",
+        "X-Enable-Telemetry": true,
```

Comment:
@mohdashraf010897 I asked you to not set the `X-Enable-Telemetry` header unless user set it to `false`. You can also use the lower case, since headers are case-insensitive.